standalone/src/main/java/com/zegelin/cassandra/exporter/Application.java

```diff
@@ -78,6 +77,9 @@ protected int defaultPort() {
 
     @Option(names = "--cql-password", paramLabel = "PASSWORD", description = "CQL authentication password.")
     private String cqlPassword;
+    
+    @Option(names = "--cql-ssl", paramLabel = "PASSWORD", description = "Native connection to be encrypted")
```

Comment:
The `paramLabel = "PASSWORD"` isn't appropriate for this option. Also the description could be improved.

Reply:
I will correct it in my next commit.